README.md

```diff
@@ -7,16 +7,38 @@ Docker: Believe the Hype!
 Learn how to use docker.io containers to consistently deploy
 your apps on any infrastructure.
 
+## Why?
+Have you ever struggled to get your developer environment set up as a new user on an existing project? Or have you ever had difficulty reproducing a bug or error that another team member is experiencing? If you have, then you have encountered some of the difficulties of not working in the same environment as your teammates on a project. Docker is the solution to these problems as well as offering a host of other benefits.
 
-## What? (*Quick* Introduction)
+Using Docker has advantages similar to those of running a virtual machine but containers are more lightweight and have less overhead/hassle.
```

Comment:
Good addition 👍 but I might change this to:

Docker containers offer the isolated environment advantages of running a virtual machine, but are more lightweight, portable, scalable and have less overhead/hassle.